web/src/components/login-form.js

```diff
@@ -1,11 +1,33 @@
 import React, {Component} from 'react';
 import PropTypes from 'prop-types';
+import {GoogleLogin} from 'react-google-login';
+import { clientID } from './utils.js'
 
 class LoginForm extends Component {
+
+    constructor(props) {
+        super(props);
+    }
+
+    // Google login success callback function
```

Comment:
Comments should start with a verb and should be a complete sentence. Same below.